include/aspect/particle/property/interface.h

```diff
@@ -329,6 +329,37 @@ namespace aspect
           initialize_one_particle_property (const Point<dim> &position,
                                             std::vector<double> &particle_properties) const;
 
+          /**
+           * Update function. This function is called every time an update is
+           * request by need_update() for every particle for every property.
+           * It is obvious that
+           * this function is called a lot, so its code should be efficient.
+           * The interface provides a default implementation that does nothing,
+           * therefore derived plugins that do not require an update do not
+           * need to implement this function.
+           *
+           * @param [in] data_position An unsigned integer that denotes which
```

Comment:
can you mention that the location can be accessed using ``particle->get_location()`` and its properties with ``particle->get_properties()``?